src/d/actor/d_a_tori_flag.cpp

```diff
@@ -4,66 +4,201 @@
 //
 
 #include "d/actor/d_a_tori_flag.h"
+#include "dolphin/mtx/mtx.h"
 #include "d/d_procname.h"
+#include "d/d_kankyo_wether.h"
+#include "m_Do/m_Do_mtx.h"
+#include "d/d_com_inf_game.h"
+#include "m_Do/m_Do_ext.h"
+#include "d/res/res_trflag.h"
+#include "d/res/res_cloth.h"
+#include "f_op/f_op_actor_mng.h"
+#include "d/d_a_obj.h"
+#include "SSystem/SComponent/c_math.h"
+
+#include "weak_bss_936_to_1036.h" // IWYU pragma: keep
+
+static dCcD_SrcCyl l_cyl_src = {
+    // dCcD_SrcGObjInf
+    {
+        /* Flags             */ 0,
+        /* SrcObjAt  Type    */ 0,
+        /* SrcObjAt  Atp     */ 0,
+        /* SrcObjAt  SPrm    */ 0,
+        /* SrcObjTg  Type    */ ~(AT_TYPE_BOOMERANG),
+        /* SrcObjTg  SPrm    */ cCcD_TgSPrm_Set_e | cCcD_TgSPrm_IsEnemy_e,
+        /* SrcObjCo  SPrm    */ cCcD_CoSPrm_Set_e | cCcD_CoSPrm_IsOther_e | cCcD_CoSPrm_VsEnemy_e | cCcD_CoSPrm_VsPlayer_e | cCcD_CoSPrm_VsOther_e,
+        /* SrcGObjAt Se      */ 0,
+        /* SrcGObjAt HitMark */ 0,
+        /* SrcGObjAt Spl     */ 0,
+        /* SrcGObjAt Mtrl    */ 0,
+        /* SrcGObjAt SPrm    */ 0,
+        /* SrcGObjTg Se      */ 0,
+        /* SrcGObjTg HitMark */ 0,
+        /* SrcGObjTg Spl     */ 0,
+        /* SrcGObjTg Mtrl    */ 0,
+        /* SrcGObjTg SPrm    */ cCcD_TgSPrm_Set_e | cCcD_TgSPrm_IsEnemy_e | cCcD_TgSPrm_IsPlayer_e,
+        /* SrcGObjCo SPrm    */ 0,
+    },
+    // cM3dGCylS
+    {
+        /* Center */ 0.0f, 0.0f, 0.0f,
+        /* Radius */ 30.0f,
+        /* Height */ 400.0f,
+    },
+};
 
 /* 000000EC-00000118       .text __ct__17daTori_Flag_HIO_cFv */
 daTori_Flag_HIO_c::daTori_Flag_HIO_c() {
-    /* Nonmatching */
+    flag = -0x1;
```

Comment:
```suggestion
    mNo = -0x1;
```
The first field of HIO classes (this s8 that defaults to -1) always seems to be the HIO child index - which TP debug strings indicate was named something like `mNo`, so I like to name them the same thing in TWW (even though it's not really used outside of debug builds)